config-toml.md

```diff
@@ -0,0 +1,40 @@
+# Config.toml breakdown
+
+- [Config.toml breakdown](#configtoml-breakdown)
+  - [Pre-Requisites](#pre-requisites)
+  - [Understanding config.toml](#understanding-configtoml)
+    - [[Core]](#core)
+    - [[P2P]](#p2p)
+      - [Bootstrap](#bootstrap)
+      - [Mutual Peers](#mutual-peers)
+    - [[Services]](#services)
+      - [TrustedHash and TrustedPeer](#trustedhash-and-trustedpeer)
+
+## Pre-Requisites
+Please, make sure that you have installed and initialized celestia node
+
+## Understanding config.toml
+After initialization, for any type of node, you will find a `config.toml` in path(default location)
+- `~/.celestia-bridge/config.toml` for bridge node
+- `~/.celestia-light/config.toml` for light node 
+
+Let's break down some of the most used sections
+### [Core]
+This section is needed for the Celestia Bridge Node. By default, `Remote = false`. Still for devnet, we are going to use the remote core option and this can also be set
+by the command line flag `--core.remote` 
+
+### [P2P]
+#### Bootstrap
+By default, the `Bootstrapper = false` and the `BootstrapPeers` is empty. If you want your node to be a bootstrapper, then activate `Bootstrapper = true`.
+BootstrapPeers are hardcoded into the node during initialisation, if no manual are added in the config file. If you want to add your own manually, you need to add the multiaddresses of the peers
+
+#### Mutual Peers
+The purpose of this config is to set up a bidirectional communication. This is usually the case for Celestia Bridge Nodes. In addition, you need to change the field 
+`PeerExchange` from false to true
+
+### [Services]
+#### TrustedHash and TrustedPeer
+`TrustedHash` is needed to properly initialize Celestia Bridge Node with the active `Remote` Core Client. Celestia Light Node will take genesis hash as the trusted one, if no hash is manually provided during initialization phase.
+
+`TrustedPeers` is the array of Bridge Nodes' peers that Celestia Light Node trusts. By default, this is an empty array as the Celestia Light Node trusts no one and syncs from genesis to latest block, if the `TrustedHash` is empty, too.
```

Comment:
```suggestion
`TrustedPeers` is a list of one or more Bridge Nodes that a Celestia Light Node can "trust" to give it block headers. By default, this is an empty array as the Celestia Light Node trusts no one and syncs from genesis to latest block, if the `TrustedHash` is empty, too.
```

> By default, this is an empty array as the Celestia Light Node trusts no one and syncs from genesis to latest block, if the `TrustedHash` is empty, too.

Is this true? I don't think it's possible for a light node to request headers if it has no trusted peer as this code is inside of `performRequest`

```go
	if len(ex.trustedPeers) == 0 {
		return nil, fmt.Errorf("no trusted peers")
	}
```

Reply:
This eventually *will* be true but since these docs are for now, then it will be confusing.

Reply:
Ok, so now, bootstrap peers are becoming trusted peers if none are manually provided. I totally forgot that point, so changing this line for Celestia Light Node